kuiper-game/Domain/Location.cs

```diff
@@ -1,26 +1,52 @@
+using System;
 using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
 
 namespace Kuiper.Domain
 {
     public static class Locations
     {
         
-        public static Location Luna = new Location("Luna", 384400, new List<Location>());
-        public static Location Earth = new Location("Earth", 150740000, new List<Location>() { Luna });
-        public static IEnumerable<Location> Destinations = new List<Location>() { Earth, Luna};
+        public static Location Luna = new Location("Luna", 384400, new List<Location>(), SatelliteType.Moon);
```

Comment:
Instead of providing a new array every time. You can use `Enumerable.Empty<Location>()` which is a static